tests/calculation.test.js

```diff
@@ -0,0 +1,38 @@
+var describe = require('mocha').describe
+var it = require('mocha').it
+var expect = require('chai').expect
+var strictEqual = require('../calculation')
+
+describe('strictEqual', () => {
+  it('return a value of true, if numbers are strictly equal to one another', () => {
+    var a = 5
+    var b = 5
+
+    let result = strictEqual(a, b)
```

Comment:
Once you get comfortable here is an even shorter way to write the same test

it('returns true for 5 and 5', () => {
    expect(strictEqual(5, 5)).to.equal(true)
  })